src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age
+    && (<p className="Person__age">{`I am ${[person.age]}`}</p>)}
```

Comment:
add a line between tags for better readability (between conditions) like

```
<h2 className="Person__name">{`My name is ${person.name}`}</h2>
    
{person.age && (
  <p className="Person__age">{`I am ${[person.age]}`}</p>
)}
```

also pay attention to the formmating like in the example: brackets `)}` should be on the next line